frontend/appflowy_flutter/lib/plugins/database_view/grid/application/grid_bloc.dart

```diff
@@ -35,6 +35,18 @@ class GridBloc extends Bloc<GridEvent, GridState> {
             );
             await rowService.deleteRow(rowInfo.rowPB.id);
           },
+          moveRow: (int from, int to) {
+            final List<RowInfo> rows = [];
+            rows.addAll(state.rowInfos);
```

Comment:
```suggestion
            final rows = [...state.rowInfos];
```

Reply:
@Xazin Is there a reason we're continuing to use addAll here? Any concerns?

Reply:
Sure I changed it, must have forgot to commit.

Reply:
Committed